src/addon/client.cpp

```diff
@@ -112,6 +112,162 @@ void addons_client::connect()
 			   << " supports: " << utils::join(server_capabilities_, " ");
 }
 
+int addons_client::get_addon_count()
```

Comment:
I think this function and the one below is now unused?

Reply:
Yep, removed.